### PR TITLE
vstart.sh: print newline character after vstart.sh help text

### PR DESCRIPTION
**Pull request by nmshelke**
Without this newline character the help text messes up the
expected position of the prompt string.

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Review by vshankar — APPROVED** [review]

**Comment by vshankar** [comment]
jenkins retest this please

**Comment by vshankar** [comment]
@nmshelke mind rebaing and pushing an update? (there's this annoying issue with some unrelated failures)

File: src/vstart.sh [hunk 1]
```diff
@@ -246,7 +246,7 @@ options:
 	--no-parallel: dont start all OSDs in parallel
 	--jaeger: use jaegertracing for tracing
 	--seastore-devs: comma-separated list of blockdevs to use for seastore
-        --seastore-secondary-des: comma-separated list of secondary blockdevs to use for seastore
+        --seastore-secondary-des: comma-separated list of secondary blockdevs to use for seastore\n
```
Comment (wjwithagen) [inline src/vstart.sh:249]: Why not put the `\n` on al line of its own.
Makes it less hidden/obscure, and easier to add a new option.

**Comment by kotreshhr** [comment]
jenkins test api

File: src/vstart.sh (hunk 1 above)
Comment (nmshelke) [reply src/vstart.sh]: Done. :)

File: src/vstart.sh [hunk 2]
```diff
@@ -247,6 +247,7 @@ options:
 	--jaeger: use jaegertracing for tracing
 	--seastore-devs: comma-separated list of blockdevs to use for seastore
         --seastore-secondary-des: comma-separated list of secondary blockdevs to use for seastore
```
Comment (wjwithagen) [inline src/vstart.sh:249]: Outline in GitHub does look shifted. Not clear is that is due to `tab` versus `space`.
But I would expect it to align??

Comment (vshankar) [reply src/vstart.sh:249]: yeah, that line has spaces (others has tab). @nmshelke care to quickly fix it?

Comment (nmshelke) [reply src/vstart.sh]: Done.

**Review by wjwithagen — APPROVED** [review]
@nmshelke 
Thanx for fixing the other odd ones as well.

**Pull request merged**